meta-lmp-base/recipes-multimedia/alsa/alsa-utils_%.bbappend

```diff
@@ -10,8 +10,11 @@ EXTRA_OECONF:append = " --with-init-asound-state-dir=${prefix}/lib/alsa"
 
 do_install:append() {
     if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
+        install -d ${D}${nonarch_libdir}/tmpfiles.d
```

Comment:
install -D is already responsible for creating the leading dirs, so no need for this extra line.

Reply:
Fixed.